src/editor/toolbar/Toolbar.js

```diff
@@ -8,6 +8,7 @@ import Col from "react-bootstrap/Col";
 import Row from "react-bootstrap/Row";
 import Form from 'react-bootstrap/Form';
 import '../Editor.css'
+import {Modal} from "react-bootstrap";
```

Comment:
Da du hier ein neues react-bootstrap element importiert hast, hättest du auch gleich die anderen imports aufräumen können.
```suggestion
import { Modal, ButtonToolbar, DropdownButton, ButtonGroup, Row, Form } from 'react-bootstrap';
```